question3/README.md

```diff
@@ -5,5 +5,5 @@
 Also, add the option -f to fold upper and lower case together, so that case distinctions are not made during sorting; for example, a and A compare equal.
 
 Compile Steps:
-
+gcc q333.c
```

Comment:
no examples in README; -2
Seems like it's working correctly though from my testing.
18/20